.circleci/config.yml

```diff
@@ -136,8 +136,15 @@ jobs:
       - install_js:
           react-version: << parameters.react-version >>
       - run:
-          name: 'Prettier'
-          command: pnpm prettier --check
+          name: '`pnpm prettier` changes committed?'
+          command: |
+            # #target-branch-reference
+            if [[ $(git diff --name-status master | grep pnpm-lock) == "" ]];
+            then
+                pnpm prettier --check
+            else
+                pnpm prettier:check
```

Comment:
Why only one of these runs as an npm script? Let's make it consistent.
I'd slightly prefer calling prettier directly (so `prettier --check . --ignore-path .lintignore`) as it's only used in CI and not really meant to be executed by developers. This will keep the scripts list more manageable.

Reply:
Ok, updated.